static/lang/kingmaker-en.json

```diff
@@ -319,5 +319,141 @@
         }
+    },
+    "PF2E.Warfare": {
+        "Army": {
+            "routThreshold": "Rout Threshold",
+            "scouting": "Scouting",
+            "morale": "Morale",
+            "maneuver": "Maneuver",
+            "maxTactics": "Max Tactics",
+            "consumption": "Consumption",
+            "standardDC": "Standard DC",
+            "recruitmentDC": "Recruitment DC",
+            "ammunition": "Ammunition",
+            "ammunitionMax": "Max Ammunition"
+        },
+        "ArmySheet": {
+            "newArmyName": "New Army",
+            "attributesHeader": "Attributes",
+            "strikesHeader": "Strikes",
+            "upgradesHeader": "Potency",
+            "routThresholdExceeded": "Army's HP is below Rout Threshold!",
+            "placeholderTacticsText": "Drag tactics here to add!",
+            "Strikes": {
+                "melee": "Melee Strike",
+                "ranged": "Ranged Strike",
+                "proficiency": "Proficiency",
+                "potency": "Potency",
+                "ammunitionLabel": "Ammunition: "
+            },
+            "ToggleLock": {
+                "lockSheet": "Lock Input Fields",
+                "unlockSheet": "Unlock Input Fields",
+                "lockWeapon": "Weapon is available",
+                "unlockWeapon": "Weapon is unavailable"
+            },
+            "InfoButton": {
+                "tooltipPotion" : "Potions Info",
+                "tooltipRanged" : "Ranged Weapons Info",
+                "tooltipMelee" : "Melee Weapons Info",
+                "tooltipArmour" : "Armour Info",
+                "levelLabel": "Level: ",
+                "priceLabel": "Price: ",
+                "resourceLabel": " RP"
+            },
+            "Potions": {
+                "label": "Potion(s)",
+                "drinkTooltip": "Use Potion (+1 HP)",
+                "drinkErrorNoPotions": "No potions available",
+                "drinkErrorFullHP": "HP is already full"
+            },
+            "StatGenerator": {
+                "title": "Generate Army Stats",
+                "desc": "Generates stats for armies using the default values as defined in the Warfare rules. New statistics will be dependent on the level. Can be used for creating new armies or for leveling up existing ones. Gear, tactics, actions, and traits will not be replaced.",
+                "warning": "Warning: The old statistics will be permanently overwritten.",
+                "parametersHeader": "Parameters",
+                "levelLabel": "Level: ",
+                "saveLabel": "Strong save: ",
+                "confirmButton": "Update Stats",
+                "levelUpTitle": "Level Up Army",
+                "levelUpDesc": "Increases Army level by 1, updating stats using the default values as defined in the Warfare rules. New statistics will be dependent on the level. Gear, tactics, actions, and traits will not be replaced."
+            }
+        },
+        "Traits": {
```

Comment:
Move these traits to `PF2E.Kingmaker.Trait`, and create a new `PF2E.Kingmaker.TraitDescription` object. `armyDescription` would then be `PF2E.Kingmaker.TraitDescription.army` instead.